src/pages/Homepage/Homepage.js

```diff
@@ -1,15 +1,25 @@
+import { useEffect } from "react";
 import { Navbar, NoteCard, Sidebar } from "../../components/index";
+import { useAuth, useNote } from "../../context";
+import { getNotes } from "../../util";
 import "./Homepage.css";
 const Homepage = () => {
+  const { authState } = useAuth();
+  const { token } = authState;
+  const { noteState, noteDispatch } = useNote();
+  useEffect(() => getNotes({ token, noteDispatch }), [authState]);
+
   return (
     <>
       <Navbar />
       <div className="homepage-container">
         <Sidebar />
         <div className="note-card-container">
-          <NoteCard />
-          <NoteCard />
-          <NoteCard />
+          {noteState.length !== 0
+            ? noteState.map(({ note, createdAt }) => (
+                <NoteCard note={note} createdAt={createdAt} />
+              ))
+            : null}
```

Comment:
Here when you have no notes then instead of showing nothing, you can display a msg like Make your first note. Because when user logged in, homepage looks empty